src/components/Marketplace/InventoryManagerScreen.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import ListingsView from './ListingsView';
+import AddListingButton from './AddListingButton';
+
```

Comment:
Could you include this makeStyle:

const useStyles = makeStyles({
  root: {
    position: 'relative',
    minHeight: '100vh',
  },
});